www/components/core/layout/FullPageBackground.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from "@emotion/styled";
+import React, { ReactNode } from "react";
+
+import { flexColumnCenter } from "./helpers";
+
+const Container = styled.div`
+  ${flexColumnCenter}
+
```

Comment:
For my education: why the empty lines?

Reply:
It's OCD mostly. I like to group my CSS into categories like
- General layout (e.g. flex, how the box should display, etc)
- High level modifiers (width, padding, margin, etc)
- Content modifiers (font, text size, etc)
- All the rest

Easier to find things when debugging huge CSS bits.

Reply:
Sounds good. I don't have a nuanced enough understanding of CSS to actually think of it in these terms, nice.